### PR TITLE
Remove twin.macro, use pure tailwindcss3

### PR DESCRIPTION
**Pull request by B2o5T**
Plz run the app to see that nothing was broken

### CODE REVIEW

File: packages/web/app/src/components/common/Navigation.tsx [hunk 1]
```diff
@@ -212,7 +211,6 @@ export function Navigation() {
             </div>
           </div>
           <ThemeButton />
-          <div className="hive-release-notes" />
```
Comment (B2o5T) [inline packages/web/app/src/components/common/Navigation.tsx:215]: does this class is somewhere used?

File: packages/web/app/src/components/project/migration-toast.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export function ProjectMigrationToast({ projectId, orgId }: { projectId: string;
         href={`/${orgId}/${projectId}/view/settings`}
         variant="link"
         size="small"
-        className="shrink-1 mr-2 self-center text-sm font-medium text-gray-300"
```
Comment (B2o5T) [inline packages/web/app/src/components/project/migration-toast.tsx:22]: `shrink-1` not exist, seems typo

**Review by enisdenjo — APPROVED** [review]

File: packages/web/app/src/components/common/Navigation.tsx (hunk 1 above)
Comment (dotansimha) [reply packages/web/app/src/components/common/Navigation.tsx:215]: I think it's unused?

**Comment by dotansimha** [comment]
@B2o5T my impression so far:

### `Create Project` does not work

This button on the main org page:
image

is no longer functional; I'm not able to click on it at all. 

### Modals position

image

image

image

seems a bit off, no longer centered 

 ### CDN `Connect` button is not working 

image

can't click at all 

> I tested all other screens and it looks the same as before ✅

**Review by dotansimha — CHANGES_REQUESTED** [review]
Please refer to my comment above.

**Pull request merged**